### DISCUSSION TITLE
Unable to build parallel with system lapack (mac os)

### DISCUSSION THREAD

**Discussion by MRR71** [Homebrew/discussions#894] [Everyday usage]
Hello everyone, 
I am trying to build lapack on my M1 Mac using $make PLATFORM=gfortran p_sys_lapack$ command. But it shows error (atatched). The procedure I followed is as follows:

1. $brew install openblas$
2. $brew install lapack$
3. $brew install gcc$
4. $brew install mpich$
5. $make PLATFORM=gfortran p_sys_lapack$ 

While steps 1-4 works perfectly fine, step 5 doesn't work. Any help will be highly appreciated.
Thank you.

![builderror](https://user-images.githubusercontent.com/45832744/108818496-63441700-75b1-11eb-9070-197de5b5a110.png)

**Comment by SMillerDev** [comment]
This needs to be reported to lapack, unless you have a specific reason to believe it's a brew error.

**Comment by jonchang** [comment]
`.s` files are assembly, so it's likely that this software hasn't been ported to ARM. You need to report this to the upstream vendor.

**Reply by MRR71** [reply]
thank you
